### PR TITLE
feat: pass location param as an input in the form

### PR DESCRIPTION
**Pull request by bolmaster2**
Passes location param provided from the `DreamsSDK.start` method in the form as a hidden input. So it gets sent to dreams-enterprise via the `/verify-token` request where it had support for handling a `location` param here: https://github.com/getdreams/dreams-enterprise/blob/main/app/controllers/users_controller.rb#L52

I had problems with the `iframe.postMessage` approach. But maybe that's a better way to go. At this point I'm too confused about this whole thing to take a good decision.

Thought: Should we help the users which `location` params they can use? We could be stricter and provide a list of values to use, e.g., an array with paths? Or we just allow them to pass anything and we don't need to update the list 🤷

### CODE REVIEW

File: docs/.nojekyll [hunk 1]
```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```
Comment (beniutek) [inline docs/.nojekyll:1]: is this necessary?

File: src/main.ts [hunk 2]
```diff
@@ -66,6 +67,12 @@ export class DreamsSDK {
 
     if (localeInput) localeInput.setAttribute('value', locale);
 
+    const locationInput: HTMLInputElement = (this.form.querySelector(
```
Comment (beniutek) [inline src/main.ts:70]: codeclimate complains about double quotes. I don't agree with that but I guess we should stick to what's been set up for other projects

File: src/main.ts [hunk 3]
```diff
@@ -50,10 +50,11 @@ export class DreamsSDK {
   }
 
   /**
-   * @param token  jwk token for the user
+   * @param token jwk token for the user
    * @param locale determines the localisation configuration that will be applied.
+   * @param location path to which the user will be redirected to after the token is verified
    */
-  start(token: string, locale: string) {
+  start(token: string, locale: string, location?: string) {
```
Comment (beniutek) [inline src/main.ts:57]: IMO you can disregard this message (or try to improve the code if it'sworth it)

File: docs/.nojekyll (hunk 1 above)
Comment (beniutek) [reply docs/.nojekyll:1]: hmmm, i think `yarn docs` is not correct we probably should add `typedoc --githubPages false` as a `yarn docs` package

Comment (bolmaster2) [reply docs/.nojekyll:1]: I don't know. It got generated by typedoc so if we don't need it we should not generate it by turning it off by that `githubPages` setting? Where are those settings?

File: src/main.ts (hunk 3 above)
Comment (bolmaster2) [reply src/main.ts:57]: Which message do you mean?

File: src/main.ts (hunk 2 above)
Comment (bolmaster2) [reply src/main.ts:70]: Where is codeclimate running?

Comment (beniutek) [reply src/main.ts:70]: do you have access to it? like to this link for example: https://codeclimate.com/repos/61487d133aebbd01a000b141

File: src/main.ts (hunk 3 above)
Comment (beniutek) [reply src/main.ts:57]: that's the code climate complaint about code complexity

File: src/main.ts (hunk 2 above)
Comment (bolmaster2) [reply src/main.ts:70]: I unfortunately don't have access to that :(

Comment (loffek) [reply src/main.ts:70]: You should be able to login with your github account?

**Review by loffek — CHANGES_REQUESTED** [review]
Nice! 

> I had problems with the iframe.postMessage approach

Can you elaborate?

File: src/main.ts (hunk 2 above)
Comment (loffek) [reply src/main.ts:70]: You should be able to login with your github account?

File: docs/.nojekyll (hunk 1 above)
Comment (bolmaster2) [reply docs/.nojekyll]: `typedoc --githubPages false` added ✅

**Review by loffek — APPROVED** [review]
Nice!

File: package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "test": "jest",
     "semantic-release": "semantic-release",
     "semantic-release-test": "semantic-release --dry-run --debug",
-    "docs": "typedoc src/*"
+    "docs": "typedoc --githubPages false src/*"
```
[the remark this answers is not in the document]
Comment (bolmaster2) [reply package.json:39]: It's as, I understand it, saying we don't use github pages and therefore a `.nojekyll` is not needed. And as we don't use github pages I think it's valid

File: src/main.ts (hunk 2 above)
Comment (bolmaster2) [reply src/main.ts:70]: I get access denied from codeclimate. But I can run it locally, which is not optimal, but it works. I have a memory codeclimate has a max limit on users on some plans, no?

File: package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (loffek) [reply package.json:39]: Summarizing out-of-band-discussion:
- We do use github pages
- but we don't need `--githubPages true` for it to work, and we don't want the `.nojekyll` file

**Comment by loffek** [comment]
Oooh. Now codeclimate works (for me). Does it work for you too @bolmaster2 ?

**Review by loffek — APPROVED** [review]

**Pull request merged**